workflow-templates/hqs-ci-test-pure-python.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       # Test code coverage of rust core is over 80 percent
       test_code_coverage: true
       # Run tests also on windows runners
-      windows: false
+      windows: true
```

Comment:
Will activating these by default break things? If so, this should be announced somewhere (same for other changed defaults)

Reply:
Same reasoning, windows hasn't shown big problems. Also, since it's simply supposed to be copy-pasted, in case something goes wrong choosing the best inputs is up to the maintainer. But on that note, we could think about setting `macos` is false by default, considering the problems that still show up every now and then.